stories/dashboard.stories.js

```diff
@@ -30,7 +30,7 @@ storiesOf( 'Dashboard', module )
 		<DashboardModuleHeader
 			title={ __( 'Module Header', 'google-site-kit' ) }
 			description={ __( 'Description of Module', 'google-site-kit' ) }
-			timePeriod={ __( 'Last 28 Days', 'google-site-kit' ) }
+			timePeriod={ __( 'Last 28 days', 'google-site-kit' ) }
```

Comment:
I think we might want to keep title case here.

Reply:
For this I've followed the pattern for the majority of similar strings:
https://github.com/google/site-kit-wp/blob/b225df85d49c114e161591c91bbdb6a3c8015995/assets/js/components/date-range-selector.js#L90

Reply:
Still, this will probably be blown away by this future enhancement #447

Reply:
Makes sense, thanks for clarifying. I like the attention to detail! :-)